packages/scaffold-config/src/frameworks.ts

```diff
@@ -122,6 +123,7 @@ export const WIZARD_FRAMEWORKS = [
     codeGenFramework: 'react',
     glob: '*.{js,jsx,tsx}',
     mountModule: 'cypress/react',
+    componentIndexHtml: componentIndexHtmlGenerator(),
```

Comment:
I like this approach. It seems like there's duplication (there is) but I think it's actually somewhat ideal.  That said, if you forgot to add `componentIndexHtml`, the only way we'd know is via a type error.

One thing we could do is add a test that loops each of these, asserting the output, like

```
for (const framework of WIZARD_FRAMEWORKS) {
  const actual = framework.componentIndexHtmlGenerator()
  if (framework === 'nextjs') {
    // specific assertion
  } else {
    expect(actual).to.eq(<generic_template>)
  }
}
```

Do you think this would be useful to have, or the types are enough to keep us safe and confident? I could go either way.

Reply:
I think that relying on types is fine - if they're not here to catch issues like this, then why do we even have them? It's also implicitly covered by the system tests (though I wouldn't want to rely on them to cover this feature, they are a backstop against the types as well).